Mod2/Labs/Week5/DatabaseMigrations.md

```diff
@@ -0,0 +1,26 @@
+## Lab Objectives
+* Practice using database migrations to add a table
+* Practice using database migrations to add a column
+* Practice creating seed data
+
+In today's lab, we're going to add to our library application and database. Before beginning make sure you have the library application open.
+
+## Practice
+
+### Adding a Table
+Your first goal is to add a new table called `events`, this table will not have an association with the `books` or `patrons` tables. You get to decide what fields make sense for your events table and what type of data your database columns should store. Feel free to use the events for the Denver library for inspiration: https://www.denverlibrary.org/events/upcoming
```

Comment:
Love that we are providing some event inspiration!

Reply:
Side motive of sharing how awesome libraries are 😄